lib/engine/game/g_18_oe/game.rb

```diff
@@ -231,6 +457,14 @@ def metropolis_tile?(tile)
              OE18 OE26 OE27 OE28 OE29 OE30 OE37 OE38 OE39 OE40 OE41].include?(tile.name.to_s)
         end
 
+        def must_buy_train?(entity)
+          # must buy the reserved 2+2, otherwise only majors must buy trains
+          # return unless entity.type == 'major'
+          return if depot.depot_trains.first&.name != '2+2' && entity.type != :major
```

Comment:
Code style: `must_buy_train?` should always return a boolean value, so this should be `return false if…`.